src/h5web/visualizations/shared/AxisSystem.module.css

```diff
@@ -44,3 +44,12 @@
   font-size: inherit;
   font-weight: 600;
 }
+
+.title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  grid-area: top-axis;
```

Comment:
I make use of the `top-axis` space. We might want to make a dedicated space for the title. 

But for now, it doesn't conflict with anything as we do not support other axis placements than left+bottom.